src/fragments/lib/graphqlapi/js/optimistic-ui.mdx

```diff
@@ -1461,7 +1521,8 @@ function App() {
                 "Updating Real Estate Property..."
               ) : (
                 <>
-                  {updateMutation.isError ? (
+                  {updateMutation.isError &&
+                  updateMutation.error instanceof Error ? (
```

Comment:
For these and the next 2 - is it possible for `isError` to be true end `error` to be undefined and or not an instance of `Error`?

Reply:
This was the suggested approach from the TanStack docs for dealing with the resulting TypeScript errors: https://tanstack.com/query/latest/docs/react/typescript#typing-the-error-field.

Reply:
Makes sense. Should `deleteMutation` and `createMutation` below follow the same pattern then?

Reply:
@iartemiev - The TS examples for `deleteMutation` and `createMutation` currently follow this pattern, as well.